spell/psm-checklists.md

```diff
@@ -0,0 +1,137 @@
+            - [ ] `_litePsm` matches `DssLitePsm` address
+            - [ ] `_rushThreshold` matches the Exec Sheet (it might be named as "fill threshold")
+            - [ ] `_gushThreshold` matches the Exec Sheet (it might be named as "trim threshold")
+            - [ ] `_cutThreshold` matches the Exec Sheet (it might be named as "chug threshold")
+- Initialization
+    - `DssLitePsm`:
+        - [ ] `buf` is set to the value specified in the Exec Sheet
+        - [ ] `vow` is set to `MCD_VOW` from the chainlog
+        - [ ] `MCD_PAUSE_PROXY` is whitelisted to execute swaps with no fees (i.e. `KissLike(LITE_PSM).kiss(MCD_PAUSE_PROXY)`)
+        - [ ] `DssLitePsmMom` is authed (i.e. `RelyLike(LITE_PSM).rely(MOM)`)
+    - `DssLitePsmMom`:
+        - [ ] The chief (`MCD_ADM`) is set as the `authority` (i.e. `MomLike(MOM).setAuthority(MCD_ADM)`)
+    - `LitePsmJob`:
+        - [ ] Job is added to `CRON_SEQUENCER`
+    - [ ] `MCD_VAT`: new ilk is initialized
+    - [ ] `MCD_JUG`: new ilk is initialized
+    - `MCD_SPOT`:
+        - [ ] Sanity check: Dai parity (`par`) current value is 1 (`1 * RAY`)
+        - [ ] Collateralization ratio (`mat`) is set to 100% (`1 * WAD`) for the ilk 
+        - IF a new `pip` is being used:
+            - [ ] `pip` is set for the ilk
+            - [ ] `pip` is a `DSValue` instance
+            - [ ] The value on `pip` is set to 1 (`1 * WAD`)
+            - [ ] `pip` deployer is no longer the `owner`
+            - [ ] `MCD_PAUSE_PROXY` is the `owner` on `pip`
+        - OTHERWISE when reusing an existing `pip`:
+            - [ ] `pip` is set for the ilk
+            - [ ] `pip` in the chainlog matches `gem` symbol (i.e. `PIP_USDC` for `USDC`)
+    - [ ] `ILK_REGISTRY`: new ilk is added to the registry
+    - `CHAINLOG`:
+        - [ ] `DssLitePsm` is added to the chainlog and `addresses_mainnet.sol` according to the Exec Sheet
+        - [ ] `pocket` is added to the chainlog and `addresses_mainnet.sol` according to the Exec Sheet
+        - [ ] `DssLitePsmMom` is added to the chainlog and `addresses_mainnet.sol` according to the Exec Sheet
+        - [ ] `LitePsmJob` is added to the chainlog and `addresses_mainnet.sol` according to the Exec Sheet
+        - [ ] IF a new `pip` is being added, it is added to the chainlog and `addresses_mainnet.sol` as `PIP_{TOKEN_SYMBOL}` (i.e. `PIP_USDC`)
+- Test coverage:
+    - `DssLitePsm`:
+        - [ ] `vow` is set to `MCD_VOW`
+        - [ ] `buf` matches the Exec Sheet
+        - [ ] `MCD_PAUSE_PROXY` is whitelisted to execute swaps with no fees (i.e. `KissLike(LITE_PSM).bud(MCD_PAUSE_PROXY) == 1`)
+        - [ ] `DssLitePsmMom` is authed (i.e. `WardsLike(LITE_PSM).wards(MOM) == 1`)
+    - [ ] `DssLitePsmMom`: chief (`MCD_ADM`) is set as the `authority` (i.e. `MomLike(MOM).authority() == MCD_ADM`)
+    - [ ] `LitePsmJob`: job is added to `CRON_SEQUENCER` (i.e.: `SequencerLike(CRON_SEQUENCER).hasJob(LITE_PSM_JOB) == true`)
```

Comment:
Don't we need to test the parameters with which it's added?

Reply:
Added the sanity checks to be peformed.